### ISSUE TITLE
Manual Mode crashing on Mac

### ISSUE THREAD

**Issue by drmaj** [deepdrive/deepdrive-sim#18]
When running [v3-mac](https://github.com/deepdrive/deepdrive-sim/tree/v3-mac/) either in the editor or standalone and without deepdrive gym attaching to deepdrive-sim. Crash report attached.
[Diagnostics.txt](https://github.com/deepdrive/deepdrive-sim/files/3406956/Diagnostics.txt)

**Comment by crizCraig** [comment]
Interesting, does this occur when the sim is started?

**Comment by drmaj** [comment]
> Interesting, does this occur when the sim is started?

Yes. Running from either the editor or on it's own. Ideally, I think the sim should work independently of the gym.

**Comment by drmaj** [comment]
The problem was here: 

https://github.com/deepdrive/deepdrive-sim/blob/46f04386ed821dbfc7a73a039717c3ad61e77832/Plugins/DeepDrivePlugin/Source/DeepDrivePlugin/Private/Simulation/DeepDriveSimulation.cpp#L68

With this logic, ScenarioMode is always true, which does not allow the sim to run in standalone mode.

**Comment by crizCraig** [comment]
@dnsmernst recently added this - can you advise @dnsmernst ?
